resources/views/service/registration.blade.php

```diff
@@ -0,0 +1,100 @@
+                </div>
+                <div class="add">
+                    <h2>
+                        <label for="sec_carers">
+                            <i class="fa fa-user" aria-hidden="true"></i> Other voucher collectors:
+                        </label>
+                    </h2>
+                    <input id="sec_carers" type="text">
+                    <button name="add-collector">
+                        <i class="fa fa-plus" aria-hidden="true"></i>
+                    </button>
+                </div>
+                <div class="collectors">
+                    <p>These people may collect vouchers:</p>
+                </div>
+            </div>
+            <div class="col">
+                <div class="add">
+                    <h2>
+                        <i class="fa fa-user-plus" aria-hidden="true"></i>
+                        Add Children <span><i class="fa fa-info-circle" aria-hidden="true"></i></span>
+                    </h2>
+                    <h3>
+                        <label for="birth_month">Month + Year of birth (or due date for pregnancy)</label>
+                    </h3>
+                    <input id="birth_month" type="month" min="1998-01">
+                    <button name="add-collector">
+                        <i class="fa fa-plus" aria-hidden="true"></i>
+                    </button>
+                </div>
+                <div>
+                    <h2>Children signed up:</h2>
+                    <table>
+                        <tr>
+                            <td>Age</td>
+                            <td>Month/Year</td>
+                            <td>Info</td>
+                        </tr>
+                        <tr>
+                            <td>7</td>
+                            <td>Sep 09</td>
+                            <td>At School</td>
+                        </tr>
+                        <tr>
+                            <td>3</td>
+                            <td>Jan 14</td>
+                            <td>On Scheme</td>
+                        </tr>
+                        <tr>
+                            <td>P</td>
+                            <td>Jun 18</td>
+                            <td>Pregnancy</td>
+                        </tr>
+                    </table>
+                </div>
+                <div class="reminder">
+                    <p>Reminder: don't forget to complete food diary and chart.</p>
+                </div>
+            </div>
+            <div class="col">
+                <div>
+                    <label for="cc-id">CC ID number:</label>
+                    <input type="text" id="cc-id">
+                </div>
+                <div>
+                    <h2>Reason for receiving Rose Vouchers <span><i class="fa fa-info-circle"
+                                                                    aria-hidden="true"></i></span></h2>
+                    <div class="user-control">
```

Comment:
Would these be better as a list?

Reply:
There will only be 2, HF or "other"; Arcane ICO/DPA reasons for not wanting custom fancylists.